src/Microsoft.AspNet.Mvc/MvcServices.cs

```diff
@@ -62,6 +62,7 @@ public static IEnumerable<IServiceDescriptor> GetDefaultServices(IConfiguration
             yield return describe.Singleton<IActionSelectorDecisionTreeProvider, ActionSelectorDecisionTreeProvider>();
             yield return describe.Scoped<IActionSelector, DefaultActionSelector>();
             yield return describe.Transient<IControllerActionArgumentBinder, DefaultControllerActionArgumentBinder>();
+            yield return describe.Singleton<IObjectModelValidator, DefaultObjectValidator>();
```

Comment:
is there a strong reason for not having it as singleton? I guess once it starts consuming services, it would make sense for it to be a transient. Will update.